src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Figure {

```

Comment:
There is an unnecessary empty line at the start of the class definition. Please remove the empty line to follow best practices.